site/www/references.md

```diff
@@ -25,6 +25,10 @@ Otherwise, we welcome you to add additional examples by [editing this page direc
 
 ## 2020
 
+* Krissaane, I, et al. "Scalability and cost-effectiveness analysis of whole genome-wide association studies on Google Cloud Platform and Amazon Web Services", *Journal of the American Medical Informatics Association* (2020) ocaa068 <https://doi.org/10.1093/jamia/ocaa068> <https://academic.oup.com/jamia/article/doi/10.1093/jamia/ocaa068/5876972>
```

Comment:
This really should be in a separate PR. Each PR should do its own thing. This is a mix of workshop stuff and references. It's fine this time, but in the future don't do this.

Reply:
Will do! So next time, workshop is one PR and a second PR for reference edits?